README.md

```diff
@@ -100,6 +100,18 @@ steps:
 - run: npm rebuild && npm run prepare --if-present
 ```
 
+Specifying a different architecture than the system architecture:
+```yaml
+steps:
+- uses: actions/checkout@master
+- uses: actions/setup-node@v1
+  with:
+    node-version: '10.x'
+    node-arch: 'x86'
```

Comment:
As this is an advanced option, I don't want the readme and straight forward example to have it.

Reply:
Many folks copy and paste to get started and installing x86 is definately a minor case.  It's ok to note further down in the readme that node-arch is an option and list it's possible values.

Reply:
I can understand that.

Reply:
I've updated the `README.md` accordingly. Let me know if you have something different in mind.